### ISSUE TITLE
Anyone managed to launch the tool in Fedora?

### ISSUE THREAD

**Issue by OmiceyO** [Screetsec/TheFatRat#587]
As the title says, did anyone here manage to launch this tool on Fedora? 
Is it possible? 
So far im missing backdoor factory and apache2(Which im not sure if it is possible because httpd)

**Comment by peterpt** [comment]
I never tested it in fedora , but backdoor factory can be directly downloaded from github and installed manually on fatrat .
for apache2 or httpd you can create a symlink between both so when fatrat calls apache2 is actually calling httpd . 
Symlink must be in one of your path variables .

Last option (not advised) is to replace everything on the scripts from apache2 to httpd .

**Comment by OmiceyO** [comment]
`whereis apache2
apache2: /usr/bin/apache2 /usr/sbin/apache2 /usr/lib/apache2 /usr/lib64/apache2 /etc/apache2 /usr/share/apache2
`
All these apache2 are symlinks which you mentioned above(Great idea), i used
 `sudo ln -s /etc/httpd/ apache2
`
in each of those directories to make those symlinks, tried to emulate it based on an ubuntu server i have.
But it didnt work, it stills doesnt detect apache2.


As for backdoor factory, i actually was trying it earlier, by cloning the directory and running the setup or install script it comes with.
I ran into some errors by trying to install it. 
So instead i tried to manually set the path to the "backdoor factory" but then it asks for for the path to searchsploit (Dont know what that is). 
Long story short it didnt work out and i posted this here. I actually cant find anyone online who installed this on a Fedora!

**Comment by peterpt** [comment]
httpd is surely not on /etc/httpd  , the httpd should be on /usr/sbin or /usr/bin or /usr/local/bin

/etc/httpd = the directory where httpd config files are .

you should look to the binary of httpd instead the directory .

searchsploit = exploit-db  , it also can be downloaded from github and installed manually on fatrat config .

**Comment by OmiceyO** [comment]
I managed to launch it!!!
I didnt generate any payload yet, but it should work!  Its possible on Fedora 33 :)
Thanks @peterpt
